test/tools/ptools/hsdinput/Br-Br.skf

```diff
@@ -0,0 +1 @@
+../../../../external/slakos/origin/3ob-3-1/Br-Br.skf
```

Comment:
Do you need symlinks here? We now have a `DFTBPLUS_PARAM_DIR` environment variable for this purpose.

Reply:
Is it possible to use the variable here? I don't call dftb+ but use it to test if the Slater-Koster files are set correctly and if reading the maximum angular momenta is possible.

https://github.com/jubich/dftbplus/blob/ddf3d9febabfbd6c78c434a1cd3ec07c33b21921/test/tools/ptools/test_hsdinput.py#L106-L132